tests/ansible_lint_rules/NoFormattingInWhenRule.py

```diff
@@ -1,20 +1,19 @@
-# Documentation on AnsibleLintRules can be found here:
```

Comment:
I assume that the file got upstreamed, so perhaps that's why the comments are gone.  Just verifying.